1-js/02-first-steps/15-function-expressions/article.md

```diff
@@ -121,24 +121,24 @@ function ask(question, yes, no) {
 */!*
 
 function showOk() {
-  alert( "You agreed." );
+  alert( "Może być." );
```

Comment:
Please replace `Może być` by `Zgoda została wyrażona`. I thought about `Zgodziłeś się` but then the expression does not have a female version. Any other ideas?